apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/live-preview-modal/style.scss

```diff
@@ -5,7 +5,7 @@
 
 .dialog__content.live-preview-modal {
 	position: relative;
-	padding: 48px 48px 28px;
+	padding: 48px 32px 28px;
```

Comment:
Sorry, can we revert this? It looks bad on Desktop, where most of live-previews will happen:

<img width="722" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/f7267198-dc73-492d-a17e-21f93dfce004">

I see there are two solutions:

1. Define this padding on a media breakpoint, or
2. Just not do it? Because I see all other modals are having the same issues 😅 e.g.

<img width="346" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1525580/95f96350-7894-418b-97d2-a300a6e57bbe">

Reply:
I'll use a breakpoint!